html/browsers/windows/nested-browsing-contexts/window-parent-null.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>window.parent: `null`</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+
+var t1 = async_test('`window.parent` is null when browsing context container element removed');
+t1.step(function () {
```

Comment:
You can use 
```js
async_test(function (t) {
 ...
 ... = t.step_func(...)
 ...
}, 'description here')
```
instead which is generally clearer then incrementing variable names.